### PR TITLE
Update k8s exporter Chart.yaml

### PR DESCRIPTION
**Pull request by yairsimantov20**
Update the k8s exporter to version v0.2.12 for the app version

### CODE REVIEW

File: charts/port-k8s-exporter/README.md [hunk 1]
```diff
@@ -53,6 +54,7 @@ The following table lists the configuration parameters of the `port-k8s-exporter
 | `createMissingRelatedEntities`          | Deprecated. Will be used only for installation, configure this from the ui / migration from older versions. A flag to enable creation of missing related Port Entities. | `false`                               |
 | `verbosity`                             | A number for the log level verbosity. Set to `-1` to print less info logs.                                                                                              | `0`                                   |
 | `createDefaultResources`                | If `true`, upon installing the exporter will try to initiate default blueprints & app configuration                                                                     | `true`                                |
+| `overwriteConfigurationOnRestart`       | If `true`, upon restarting the exporter will overwrite the configuration in port with the resources provided in `configMap.config`                                      | `false`                               |
```
Comment (MPTG94) [inline charts/port-k8s-exporter/README.md:57]:
```suggestion
| `overwriteConfigurationOnRestart`       | If `true`, the exporter will overwrite the configuration in Port with the resources provided in `configMap.config` during restart                                      | `false`                               |
```

File: charts/port-k8s-exporter/values.yaml [hunk 2]
```diff
@@ -7,13 +7,14 @@ createMissingRelatedEntities: false
...
 createDefaultResources: true
+overwriteConfigurationOnRestart: false
 
 image:
   repository: ghcr.io/port-labs/port-k8s-exporter
   pullPolicy: IfNotPresent
   tag: ""
 
-imagePullSecrets: []
+imagePullSecrets: [ ]
```
Comment (MPTG94) [inline charts/port-k8s-exporter/values.yaml:17]: Remove this change and the rest of the empty changes

**Review by MPTG94 — APPROVED** [review]
Apply fixes based on the review and you can merge

File: charts/port-k8s-exporter/values.yaml [hunk 3]
```diff
@@ -26,7 +27,7 @@ eventListener:
   securityProtocol: "SASL_SSL"
   authenticationMechanism: "SCRAM-SHA-512"
 secret:
-  annotations: {}
+  annotations: { }
```
Comment (stavbernazport) [inline charts/port-k8s-exporter/values.yaml:30]: Same whyyyyy

**Pull request merged**